### PR TITLE
NZSL-81 package and info endpoints

### PR DESCRIPTION
**Pull request by nora-ns**
Copy the `package` and `info` endpoints from Global Signbank and adjust to work for NZSL.

Visually the current results match the responses from Global Signbank. 

No usable tests have been available to copy over and due to time constraints they have not been included in this PR.

### CODE REVIEW

File: signbank/dictionary/models.py [hunk 1]
```diff
@@ -599,6 +601,110 @@ class Meta:
...
+        It has been adapted to work for NZSL's data structure.
+        """
+
+        # TO DO include other gloss relations in the fieldnames (e.g., simultaneous morphology, below)
+        # a way to determine w/o hard-coding if a fieldname is for a related model
+        # then can use the above display methods
+        # related_models = [rel.related_model._meta.model_name for rel in self._meta.related_objects]
+        # print(related_models)
+        # requires changing search field names
+
```
Comment (jonholdsworth) [inline signbank/dictionary/models.py:629]: I'm fine with future-notes in comments but others may object 🙂

File: signbank/dictionary/models.py [hunk 2]
```diff
@@ -599,6 +601,110 @@ class Meta:
...
+                fc_category = field.field_choice_category
+            else:
+                fc_category = None
+            fields_data.append((field.name, field.verbose_name.title(), fc_category))
+
+        # gloss translations
+        if self.dataset:
+            for language in self.dataset.translation_languages.all():
+                glosstranslations = self.glosstranslations_set.filter(language=language)
+                if glosstranslations and glosstranslations.count() > 0:
```
Comment (jadedarko) [inline signbank/dictionary/models.py:649]: **nitpick**: `len(query) and bool(query)` are usually better than using `.count()` or `.exists()` as they don't trigger a new query

**Review by jadedarko — COMMENTED** [review]
I'm assuming you've tried to change as little as possible from the original functions, so I'm not going to suggest much in the way of changes

It would be really good to see some tests for the endpoints if not the methods, as this is quite complex

**Review by jonholdsworth — APPROVED** [review]

**Comment by G-Rath** [comment]
@nora-ns looks like this is good for you to land and let the client know

**Review by G-Rath — APPROVED** [review]
squash when merging

**Pull request merged**